web_src/css/repo.css

```diff
@@ -1059,7 +1059,7 @@
 }
 
 .repository.view.issue .comment-list .comment .ui.form .field.footer {
-  overflow: hidden;
+  overflow: visible;
 }
```

Comment:
`visible` is the default value for `overflow`, so you can likely remove this rule.